Command.py

```diff
@@ -1,12 +1,102 @@
+from hogwartsexceptions import MaraudersMapError, RowlingError, Messages
+
+
 class Command(object):
-	''' Similar to my previous idea of having verbs except it doesn't suck'''
-	def __init__(self, syntax=None, rules=None, state_changes=None, errors=None, query=None):
-		self.syntax = syntax
-		self.rules = rules
-		self.state_changes = state_changes
-		self.errors = errors
-		self.query = query
+    """ Holds logic for, well, commands.
+    """
+    ##############################################################
+    # Example ChangefulCommand and associated methods
+    ##############################################################
+
+    # go = ChangefulCommand(syntax=[is_a_direction], rules=path_exists, state_changes=move_player)
+
+    # def is_a_direction(castle, word):
+    #     return word in castle.directions
+    #
+    # def path_exists(castle, player, direction):
+    #     if not <condition>:
+    #         raise RowlingError(Messages.NO_PATH)
+
+    # def move_player(castle, player, direction):
+    #     pass
+
+    # def look(castle, player):
+    #     pass
+
+    def __init__(self, name=None, syntax=None, rules=None, response=None):
+        self.name = name
```

Comment:
maybe rename `name` to something more indicative of what it is (a way to find the command based on something the person types). `word`? that's bad